contracts/ClaimManager.sol

```diff
@@ -109,6 +109,10 @@ contract ClaimManager is DepositsManager {
   // @param _plaintext – the plaintext blockHeader.
   // @param _blockHash – the blockHash.
   // @param claimant – the address of the Dogecoin block submitter.
+  function calcId(bytes _plaintext, bytes32 _hash, address claimant, bytes32 proposalId) public pure returns (uint) {
```

Comment:
I think this could be moved to scryptRunner where it can be computed offchain

Reply:
do we need to trust that it was computed correctly?

Reply:
Well users can pass in the id's anyways. Doesnt matter if its done in scryptVerifier.calcId or in scryptRunner. Better yet we could use web3.utils.sha3